src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -219,12 +245,22 @@ internal class ScopedHttpFunctionInvoker : IAsyncTexlFunction
         private readonly string _cacheScope;
         private readonly HttpFunctionInvoker _invoker;
 
-        public ScopedHttpFunctionInvoker(string cacheScope, HttpFunctionInvoker invoker)
+        public ScopedHttpFunctionInvoker(DPath ns, string name, FormulaType returnType, string cacheScope, HttpFunctionInvoker invoker)
         {
+            Namespace = ns;
+            Name = name;
+            ReturnType = returnType._type;
+
             _cacheScope = cacheScope;
             _invoker = invoker ?? throw new ArgumentNullException(nameof(invoker));
         }
 
+        public DPath Namespace { get; }
+
+        public string Name { get; }
+
+        public DType ReturnType { get; }
```

Comment:
>DType [](http://example.com/codeflow?start=15&length=5)

nit; FormulaType

we should minimize use of Dtype since it's internal and hopefully Connectors could eventually drop internals access.